doc/helpers.cmake

```diff
@@ -15,6 +15,21 @@ function(ginkgo_doc_conf_concat base_in in1 in2 out)
     file(WRITE "${out}" "${so}")
 endfunction()
 
+# writes the concatenated configured files <in1,2>
+# in <base_in> into <out>
+function(ginkgo_md_page_concat base_in in0 in1 in2 in3 in4 out)
+    ginkgo_configure_to_string("${base_in}/${in0}" s0)
+    ginkgo_configure_to_string("${base_in}/${in1}" s1)
+    ginkgo_configure_to_string("${base_in}/${in2}" s2)
+    ginkgo_configure_to_string("${base_in}/${in3}" s3)
+    ginkgo_configure_to_string("${base_in}/${in4}" s4)
+    set(${sep1} "@page install_ginkgo Installing Ginkgo. \n")
+    set(${sep2} "@page test_ginkgo Testing Ginkgo. \n")
+    set(${sep3} "@page benchmark_ginkgo Benchmarking Ginkgo. \n")
+    string(CONCAT so "${s0}" "\n" "${s1}" "@page install_ginkgo Installing Ginkgo. \n" "${s2}"  "@page test_ginkgo Testing Ginkgo. \n" "${s3}" "@page benchmark_ginkgo Benchmarking Ginkgo. \n" "${s4}")
```

Comment:
That also works instead of using `perl`. Quick comment: you can use `sep1` here instead of `@page install_ginkgo...` and same for the other ones.

Reply:
Yeah, I wrote those variables for that reason :) , but actually you have to convert them to a string otherwise they wont work.